tests/components/deconz/test_deconz_event.py

```diff
@@ -127,3 +128,4 @@ async def test_deconz_events(hass):
 
     assert len(hass.states.async_all()) == 0
     assert len(gateway.events) == 0
+    assert len(gateway.entities[EVENT]) == 0
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6239266)